### PR TITLE
Use more typical go structure, print ip on launch

### PR DESCRIPTION
**Pull request by calvin-brizzi**
Avoids need for relative paths. Makes it easier to get to running server.

### CODE REVIEW

File: import.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"time"
 
-	"./api"
+	"src/getrankd/api"
```
Comment (calvin-brizzi) [inline import.go]: Aw fook. Typo. No "src"

File: main.go [hunk 2]
```diff
@@ -132,6 +132,6 @@ func main() {
 
 	log.Print("Initializing API...")
 	api.Initialize()
-	log.Print("Running web server...")
+	log.Printf("Running web server at: http://127.0.0.1%s ...", server.Addr)
```
Comment (AceFire6) [inline main.go:135]: Does `server.Addr` have the colon already?

Comment (calvin-brizzi) [reply main.go:135]: Yeah, it does :+1:

Comment (jacquesh) [reply main.go:135]: Should we not just have the IP specified (as 0.0.0.0) in the server initialization? Then it'll go into the string anyways?

**Comment by AceFire6** [comment]
🔔 @calvin-brizzi

**Pull request merged**